demos/sahi/requirements.txt

```
@@ -0,0 +1,3 @@
+opencv-python==4.2.0.32
+sahi==0.10.5
```

Lets use the latest yolov5 and sahi versions :)

```suggestion
sahi==0.10.6
```

---

Awesome! I have updated it and recreated the demo video just in case!